console

```diff
@@ -42,27 +73,103 @@ function setup {
 		echo "[OK] kubectl found"
 	fi
 
+    read -p "What is the name of the context? " CONTEXT_NAME
```

Comment:
That is "kubectl-only": shouldn't we ask the name of the project (but only matching `[a-z0-9-_]+` AFAIR) ? That said, it's worth saying that we need to have `kubectl` using the correct project everytime.